services/movie_session.py

```diff
@@ -24,6 +24,15 @@ def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
     return MovieSession.objects.get(id=movie_session_id)
 
 
+def get_taken_seats(movie_session_id: int) -> list:
+    movie_session = MovieSession.objects.get(id=movie_session_id)
+    tickets = Ticket.objects.filter(movie_session=movie_session)
+    taken_seats = [
+        {"row": ticket.row, "seat": ticket.seat} for ticket in tickets
+    ]
+    return taken_seats
```

Comment:
I think you can return the result directly without creating a `taken_seats` variable